lib/log_parser.rb

```diff
@@ -19,31 +17,48 @@ def first_line
 
   def gamelog_json
     @gamelog_obj_ruby = {
```

Comment:
@perchespierri não há necessidade de atribuir o resultado a uma variável de instância da classe. Você pode retornar diretamente

```suggestion
    gamelog_obj_ruby = {
```

Reply:
Nem reparei que tinha parado de usar ela em outro lugar do código, e o @ ficou. Bem notado. Obrigado!